server/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilders.java

```diff
@@ -364,4 +365,12 @@ public static ScriptedMetricAggregationBuilder scriptedMetric(String name) {
     public static CompositeAggregationBuilder composite(String name, List<CompositeValuesSourceBuilder<?>> sources) {
         return new CompositeAggregationBuilder(name, sources);
     }
+
+    /**
+     * Create a new {@link TimeSeriesAggregationBuilder} aggregation with the given name.
+     */
+    public static TimeSeriesAggregationBuilder timeSeries(String name) {
+        return new TimeSeriesAggregationBuilder(name);
+    }
```

Comment:
Do we need to keep this class around any more? I thought it was mostly for the transport client and I've been sort of ignoring it for years.

Reply:
It is nice for IT tests that I really needed here to make sure that whole thing works. I am ok with removing it as a class, but we probably need a bigger discussion and do it outside of this PR.